docs/source/migration.mdx

```diff
@@ -403,7 +415,9 @@ interface ContextValue {
 const server = new ApolloServer({
   typeDefs,
   resolvers,
-  context: ({ req: ExpressRequest }): Omit<ContextValue, 'dataSources'> => { //highlight-line
+  //highlight-start
```

Comment:
What's up with all the highlight changes? Is `highlight-line` buggy?

Reply:
Yeah, the translation between TS -> JS with these longer lines can sometimes create a new highlighted line depending on the line length. From the authoring standpoint, I honestly think it looks cleaner using `highlight-start` and `end` on these long lines. Let me know if you disagree; this is my personal preference and by no means our standard!

Reply:
Ah cool. So we should generally avoid `highlight-line` on MultiCodeBlock?  Does DXE know about this, maybe it's fixable?

Reply:
I'd say in 90% of cases `highlight-line` works great; it's just in long lines where it can be a problem.

Reply:
[Actually, I saw Daniel push a fix for this!](https://github.com/trevorblades/remark-typescript/commit/db4bf5619d3cd07aad759a5270ecd083a640fbf8). I just merged this, and I don't think it's worth reverting this one, but good to know for the future!